### ISSUE TITLE
Buildings.Templates.Components.Validation.PumpMultipleRecord model fails during NFEvalConstants.evaluate

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#13114]
@perost please check [Buildings.Templates.Components.Validation.PumpMultipleRecord](https://libraries.openmodelica.org/branches/master/Buildings_latest/files/Buildings_latest_Buildings.Templates.Components.Validation.PumpMultipleRecord.err). The frontend fails with
```
Error: Internal error Instantiation of Buildings.Templates.Components.Validation.PumpMultipleRecord failed with no error message.
```

**Comment by Pavanrajhr** [comment]
@casella @AndreaBartolini  This model `Buildings.Templates.Components.Validation.PumpMultipleRecord` is currently passing. We can close this ticket.

**Comment by perost** [comment]
It was apparently fixed by #13313 when trying to fix another of the pump models, see [report](https://libraries.openmodelica.org/branches/history/master/2024-11-29%2011:39:24..2024-11-29%2022:38:52.html).
